modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java

```diff
@@ -113,7 +113,12 @@ public void populateConfigJSONObject(
 				CKEditorConstants.ATTRIBUTE_NAMESPACE + ":resizable"));
 
 		if (resizable) {
-			jsonObject.put("resize_dir", "vertical");
+			String resizeDirection = GetterUtil.getString(
+				inputEditorTaglibAttributes.get(
+					CKEditorConstants.ATTRIBUTE_NAMESPACE +
+						":resizeDirection"));
+
+			jsonObject.put("resizeDirection", resizeDirection);
```

Comment:
shouldn't this one still be `resize_dir`?

Reply:
You're correct. https://github.com/brianchandotcom/liferay-portal/pull/123119